compiler/rustc_macros/src/serialize.rs

```diff
@@ -120,7 +120,7 @@ fn decode_field(field: &syn::Field) -> proc_macro2::TokenStream {
     let __decoder = quote! { __decoder };
     // Use the span of the field for the method call, so
     // that backtraces will point to the field.
-    quote_spanned! {field_span=> #decode_inner_method(#__decoder) }
+    quote_spanned! { field_span => #decode_inner_method(#__decoder) }
```

Comment:
This isn't concordant with the way that the macro suggests it should be formatted: https://docs.rs/proc-quote/latest/proc_quote/macro.quote_spanned.html#syntax

> The lack of space before the => should look jarring to Rust programmers and this is intentional. The formatting is designed to be visibly off-balance and draw the eye a particular way, due to the span expression being evaluated in the context of the procedural macro and the remaining tokens being evaluated in the generated code.

Reply:
TIL

Reply:
lolwut?

I've removed the space. After that, I count five calls to `quote_unspanned` in the repo that omit the space before the `=>` as advised and five that don't. The docs also say you should use parens when the `quote_unspanned` invocation fits on one line, and braces when it doesn't -- one of the occurrences in the repo follows that advice, and nine don't. Seems like making unusual formatting suggestions in docs doesn't work that well?